src/views/OrderDetail.vue

```diff
@@ -327,7 +327,10 @@ export default defineComponent({
           translucent: true,
           showBackdrop: true,
           componentProps: { 'id': id, 'isVirtual': isVirtual, 'item': item }
-        })
+        });
+        popover.onDidDismiss().then(() => {
```

Comment:
Why these changes are done?

Reply:
When using revert only option, the data is not updated on searched product.